app/controllers/api/v1/documents_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class DocumentsController < ApiController
+      CONTENT_TYPE = 'multipart/form-data'
+
+      def create
+        document = Document.create!(document_attributes)
+        render_document(document, 201)
+      end
+
+      def destroy
+        document = Document.find(params[:id])
+        document.destroy!
+        render_document(document, 200)
+      end
+
+      private
+
+      PERMITTED_DOCUMENT_PARAMS = [
+        attributes: %i[file]
+      ].freeze
+
+      def document_params
+        params.require(:data).permit(PERMITTED_DOCUMENT_PARAMS).to_h
+      end
+
+      def document_attributes
+        document_params[:attributes].merge(
+          move: Move.find(params.dig(:move_id))
+        )
+      end
+
+      def render_document(document, status)
+        render json: document, status: status
+      end
```

Comment:
Hi @ngw, could we un-refactor this? It appears to only swap named parameters for positional ones, and IMHO the `render` call is clearer in-line.